samples/sensor/fdc2x1x/boards/nrf9160dk_nrf9160.overlay

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 arithmetics ITS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c2 {
+	clock-frequency = <I2C_BITRATE_FAST>;
+	fdc2x1x@2A                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                {
+		compatible = "ti,fdc2x1x";
+		reg = <0x2A>;
+		label = "FDC2X1X";
+		sd-gpios = <&gpio0 16 GPIO_ACTIVE_HIGH>;
+		intb-gpios = <&gpio0 20 GPIO_ACTIVE_LOW>;
+
+		// using the default channel settings
```

Comment:
Judging by this comment, I think the binding should probably not have default settings, but instead the properties should be made `required` and this sample can provide some sensible defaults for evaluation here.

Reply:
Made all channel bindings, except for `offset`, required